### PR TITLE
feat(styles): invalid message on card-control

### PR DESCRIPTION
**Pull request by davidritter-dotcom**

### CODE REVIEW

**Comment by oliverschuerch** [comment]
There is an open PR for merging the card-control stories. Please do not merge this PR before the other!

**Review by oliverschuerch — CHANGES_REQUESTED** [review]
Something looks wrong with the icon placement
image

Tested on the css-component (web-component looks good).

**Comment by davidritter-dotcom** [comment]
I've only added the icon in the message on the form elements who don't have an icon inside in this PR, but since the icon inside gets removed everywhere, the icon would need to be added everywhere in the validation message.

File: packages/styles/src/components/form-validation.scss [hunk 1]
```diff
@@ -25,16 +28,46 @@
...
+  }
+}
+
+input:not([type='radio'], [type='checkbox']) ~ .valid-feedback::before,
+textarea ~ .valid-feedback::before {
+  display: none;
+}
+
+post-card-control ~ .invalid-feedback {
+  display: inline-flex;
```
Comment (oliverschuerch) [inline packages/styles/src/components/form-validation.scss:70]: This should be part of the post-card-control.scss.

**Review by oliverschuerch — CHANGES_REQUESTED** [review]
Please add a changeset!

**Review by oliverschuerch — COMMENTED** [review]
@davidritter-dotcom did you add the error-message only for the web-component or also for the css-component?
And could we add the error-message also in the default story of both components?

**Comment by davidritter-dotcom** [comment]
I added the icon for all the invalid- and valid-feedback messages

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**